apps/api/v1/lib/validations/attendee.ts

```diff
@@ -14,7 +15,7 @@ export const schemaAttendeeBaseBodyParams = Attendee.pick({
 const schemaAttendeeCreateParams = z
   .object({
     bookingId: z.number().int(),
-    email: z.string().email(),
+    email: z.string().regex(emailRegex),
```

Comment:
We should be more declarative here like defining an `emailSchema` and importing that directly.

Reply:
Sure, I've made changes to the code by creating a new file `emailSchema.ts` that defines the `emailSchema` using the Reg Exp. This schema is now imported and used in the places where email validation is needed.